### ISSUE TITLE
Issues with Java and test cases

### ISSUE THREAD

**Issue by danielfeser** [adidas/coding-challenges#13]
Ambiguity with expected values and test case values

The height of smallest stripe, the one on the left side, is the rounded value of the square root of the stripe width.

IE. sqrt(3) = 1.73205080757.

Currently, the test case outputs for the first bar a 3x2 (wxh) whereas, when given an example in the description the first bar is 3x1 (wxh).  I have submitted a change for this within the test case only.  There are other errors where rounding would also cause mismatched testing values, ie 7.

**TLDR;**
Reduce ambiguity on rounding,  test case results in **JAVA** currently round up, whereas in the examples they round down. (java truncates, suggest rounding down and changing test case results), also fix java test cases where this is prevalent.

**Comment by moelders** [comment]
Thanks @danielfeser for raising the issue, we have to check it and to align with other languages too.

**Comment by moelders** [comment]
@santiagomorte and I were talking about the issue and it is not an issue. `java.lang.Math.round()` calculates the height as in the description:

```java
Math.round(Math.sqrt(3)) // 2
Math.round(Math.sqrt(5)) // 2
Math.round(Math.sqrt(7)) // 3
```

We do not understand why your compiler is returning `1` instead of `2` in that case.

**Comment by danielfeser** [comment]
@moelders, I was casting down to integer instead of rounding :), so that solves that problem!

But there is still ambiguity in the test cases and the example which is what I think sparked my confusion, File attached

[adidasTestCase.txt](https://github.com/adidas/coding-challenges/files/2842349/adidasTestCase.txt)

Further, _The height of smallest stripe, the one on the left side, is the rounded value of the square root of the stripe width._  So I believe the test cases are right but the example is wrong

**Comment by moelders** [comment]
OK, I understand the issue. It has been fixed: ([README.md](https://github.com/adidas/coding-challenges/tree/master/adidas-ascii-logo#example)).

Thank you @danielfeser!
